src/System.Management.Automation/security/SecuritySupport.cs

```diff
@@ -618,6 +619,17 @@ internal static bool CertIsGoodForEncryption(X509Certificate2 c)
                  CertHasKeyUsage(c, X509KeyUsageFlags.KeyEncipherment)));
         }
 
+        /// <summary>
+        /// Check to see if the specified cert is expiring in days.
+        /// </summary>
+        /// <param name="c">Certificate object.</param>
+        /// <param name="expiring">Certificate expire time.</param>
+        /// <returns>True on success, false otherwise.</returns>
+        internal static bool CertIsExpiringInDays(X509Certificate2 c, DateTime expiring)
```

Comment:
This method name is confusing to me.  Mabye: 'CertExpiresByDate(c, expireDate)'?

Reply:
Current code takes in account a time too so renamed in CertExpiresByTime().